src/poll.c

```diff
@@ -0,0 +1,356 @@
+#include <fcntl.h>
+#include <pthread.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/poll.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include "common.h"
+#include "poll.h"
+#include "log.h"
+
```

Comment:
include files are now centrally managed in `common.h`, so that all files in netdata include only `common.h`.